### PR TITLE
add file test-1.html

### PR DESCRIPTION
**Pull request by mohamedragabdahab**

### CODE REVIEW

File: test-1.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                    document.write(x + " ");
+                    break;
+                case "odd":
+                    if (x%2!=0)
+                    document.write(x + " ");
+                    break;
+                default : document.write(x + " ")
+            }
+
+
```
Comment (mohamedragabdahab) [inline test-1.html:16]: too many empty lines, please remove

File: test-1.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<script>
+    function myNumbers(start,end, option){
+        for (var x = start ; x <= end ; x++){
+            switch (option){
+                case "even":
+                        if (x%2==0)
```
Comment (mohamedragabdahab) [inline test-1.html:6]: please add if condition braces, for best practices.

File: test-1.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<script>
+    function myNumbers(start,end, option){
```
Comment (mohamedragabdahab) [inline test-1.html:2]: please find better function name

File: test-1.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+                default : document.write(x + " ")
+            }
+
+
+
+
+        }
+    }
+    myNumbers(1, 20, "even");
+</script>
```
Comment (mohamedragabdahab) [inline test-1.html:22]: general comment, please format the file, make right indentation

File: test-1.html (hunk 3 above)
Comment (mragab2000) [reply test-1.html:2]: like!!?

File: test-1.html (hunk 4 above)
Comment (mragab2000) [reply test-1.html:22]: aywa a3ml eh ya3ni

File: test-1.html (hunk 3 above)
Comment (mohamedragabdahab) [reply test-1.html:2]: printNumbersWithOptions()

File: test-1.html (hunk 4 above)
Comment (mohamedragabdahab) [reply test-1.html:19]: please search for phpstorm code formate

**Pull request closed without merging**